framework/include/utils/MooseTypes.h

```diff
@@ -186,7 +186,16 @@ enum AuxGroup
 enum VarKindType
 {
   VAR_NONLINEAR,
-  VAR_AUXILIARY
+  VAR_AUXILIARY,
+  VAR_ANY
+};
+
+enum VarFieldType
+{
+  VAR_FIELD_STANDARD,
```

Comment:
@lindsayad does this naming match up with your new naming for MooseVariable types?